charts/dapr/charts/dapr_scheduler/values.yaml

```diff
@@ -23,6 +23,7 @@ ports:
   protocol: TCP
   apiPort: 50006
   etcdRPCClientPort: 2379
+  etcdHttpClientPort: 2330
```

Comment:
Should this be `2381`?

Reply:
I considered the same, and it's open for discussion, but I thought a second scheduler instance would be using 2381 for the main (gRPC) port. So let's say if we had 5 instances, we'd have 2379, 2381, 2383, 2385 and 2387 for the client ports. That's why I chose a random port that's reasonably far away for http.

Reply:
fair enough